src/index.ts

```diff
@@ -16,9 +15,10 @@ export {
   blsCreateProof,
   verifyProof,
   blsVerifyProof,
-  commitmentForBlindSignRequest,
-  blindSign,
-  verifyBlindSignContext,
 } from "./bbsSignature";
 
-export { bbsNativeBindings } from "./register";
+const module = NativeModules.BbsSignatures;
+const res = module.install();
+if (!res) {
+  throw new Error("nope...");
```

Comment:
Can we make this a bit more descriptive?

Also should we create a custom error for this library?